docs/language_specifications.md

```diff
@@ -1,60 +1,46 @@
 # Language Specifications
 ## Assignment Statements
+```js
 let x = 31;
-
+```
 ## Conditional Statements
-if ( condition) {	}
-
-elif ( condition) { }
-
+```bash
+if   ( condition ) { }
+elif ( condition ) { }
 else { }
+```
 
 ## Conditions
-***boolean values/variables***
+[//]: <> (***boolean values/variables***)
```

Comment:
I think we should have a meeting to decide truthy/falsy values (also for declarations and constants)

Reply:
yep, but for now, I've just made them into comments